yt/testing.py

```diff
@@ -444,6 +444,51 @@ def fake_vr_orientation_test_ds(N = 96, scale=1):
     ds = load_uniform_grid(data, arr.shape, bbox=bbox)
     return ds
 
+
+def construct_octree_mask(prng=RandomState(0x1d3d3d3), refined=[True]):
```

Comment:
just in case you didn't know where this comes from:

https://www.youtube.com/watch?v=XWX4GUYGQXQ

Reply:
:laughing: good to know...
I would have kept it 0x4d3d3d3, if it had not generated a trivial mask.